core-web/libs/dotcms-scss/jsp/scss/backend/dot-admin/_dot-admin.scss

```diff
@@ -423,7 +424,7 @@ tr.active {
 .lockedAgo {
     display: block;
     color: #999;
-    font-size: 11px;
+    font-size: $font-size-xs;
     font-style: italic;
     line-height: 14px;
```

Comment:
Maybe this can use a variable

Reply:
Do you mean for the line-height? It seems not to be an standard so I think is better to keep as it is for now

Reply:
Right I agree